sulfur_simulation/hopping_calculator.py

```diff
@@ -229,19 +208,40 @@ def __init__(
         *,
         baserate: BaseRate,
         temperature: float,
-        lattice_spacing: float,
+        lattice_properties: tuple[float, np.ndarray, np.ndarray],
```

Comment:
it's not clear form the types what the different bits mean - maybe split this variable up?